help/channel-permissions.md

```diff
@@ -1,131 +1,123 @@
-| Add others            | &#10004;          | &#10038;   | &#10038;  |
-| Remove others         | &#10004;          | &#10038;   | &#10038;  | &#10038;
-| See subscriber list   | &#10004;          | &#10004;   | &#10004;  | &#9726;
-| See full history      | &#10004;          | &#10004;   | &#10004;  | &#9726;
-| See estimated traffic | &#10004;          | &#10004;   | &#10004;  | &#9726;
-| Post                  | &#10004;          | &#10038;   | &#10038;  | &#10038;
-| Change the privacy    | &#10004;          |            |           |
-| Rename                | &#10004;          |            |           |
-| Edit the description  | &#10004;          |            |           |
-| Delete                | &#10004;          |            |           |
-
-<span class="legend_symbol">&#10004;</span><span class="legend_label">Always</span>
-
-<span class="legend_symbol">&#9726;</span><span class="legend_label">If subscribed to the channel</span>
-
-<span class="legend_symbol">&#10038;</span>
-<span class="legend_label">
-Configurable. See [Channel posting policy](/help/channel-posting-policy),
-[Configure who can add users][add-users], and
-[Configure who can remove users][remove-users]
-for details.
-</span>
-
-### Private channels
-
-<div class="centered_table"></div>
-|                       | Owners and admins | Moderators | Members   | Guests
-|---                    |---                |---         |---        |---
-| View channel name     | &#10004;          | &#9726;    | &#9726;   | &#9726;
-| Join                  |                   |            |           |
-| Unsubscribe           | &#9726;           | &#9726;    | &#9726;   | &#9726;
-| Add others            | &#9726;           | &#10038;   | &#10038;  |
-| Remove others         | &#10004;          | &#10038;   | &#10038;  | &#10038;
-| See subscriber list   | &#10004;          | &#9726;    | &#9726;   | &#9726;
-| See full history      | &#10038;          | &#10038;   | &#10038;  | &#10038;
-| See estimated traffic | &#10004;          | &#9726;    | &#9726;   | &#9726;
-| Post                  | &#9726;           | &#10038;   | &#10038;  | &#10038;
-| Change the privacy    | &#9726;           |            |           |
-| Rename                | &#10004;          |            |           |
-| Edit the description  | &#10004;          |            |           |
-| Delete                | &#10004;          |            |           |
-
-<span class="legend_symbol">&#10004;</span><span class="legend_label">Always</span>
-
-<span class="legend_symbol">&#9726;</span><span class="legend_label">If subscribed to the channel</span>
-
-<span class="legend_symbol">&#10038;</span>
-<span class="legend_label">
-Configurable, but at minimum must be subscribed to the channel.
-See [Channel posting policy](/help/channel-posting-policy),
-[Configure who can add users][add-users], and
-[Configure who can remove users][remove-users]
-for details.
-</span>
+There are three types of channels in Zulip:
```

Comment:
Should "channels" be a link or something?

Reply:
Think maybe not given the channels-intro macro just before this.

Reply:
Yeah, I think not.